nuntium/templates/nuntium/profiles/contacts/contacts-per-writeitinstance.html

```diff
@@ -38,24 +38,24 @@
   <div class="page-header">
     <h2>{% trans "Recipients" %}</h2>
   </div>
-
+  <p>{% trans "There are" %} <strong>{{ writeitinstance.persons.count }}</strong> {% trans "recipients in your WriteIt" %}</p>
```

Comment:
I'm assuming there's a better way to do this that will work in languages that form a sentence like this differently? (and to cope with the case of only having one person)